ironfish/src/wallet/wallet.ts

```diff
@@ -922,16 +923,14 @@ export class Wallet {
         raw.fee = getFee(options.feeRate, raw.size())
       }
 
-      await this.fund(raw, {
-        fee: raw.fee,
-        account: options.account,
-        confirmations: confirmations,
-      })
-
-      if (options.feeRate) {
-        raw.fee = getFee(options.feeRate, raw.size())
-        raw.spends = []
-
+      if (options.spendNoteHashes) {
```

Comment:
it would be nice if fund took these notes as an optional parameter

Reply:
Implemented here: https://github.com/iron-fish/ironfish/pull/3830/commits/c146223e9046ecfdc80c63efe08db96a5f858e18